requirements.txt

```diff
@@ -2,6 +2,7 @@ backports-abc==0.5
 boto3>=1.4.7
 chardet==3.0.4
 compressed-segmentation>=1.0.0
+DracoPy>=0.0.5
```

Comment:
This is great! However, people often complain when I add additional compilation steps to CloudVolume. Would you be interested in creating binaries for DracoPy for Linux? I had trouble compiling on MacOS the other day though I didn't dig very hard.

Reply:
So I looked into creating manylinux binaries today. I had trouble building draco on manylinux/CentOS 5. I think the gcc compiler on CentOS 5 might be too old. Could we hold off on this until manylinux2010 becomes standard? It will drop CentOS 5 support. It seems relatively close: https://github.com/pypa/manylinux/issues/179

Reply:
@sdorkenw this impacts you the most, what do you think? Do we need some kind of exception logic in case DracoPy can't compile so that you can still use Cloudvolume _Graphene Collectors' Edition_ without it?